src/postgres/7-seed.sql

```diff
@@ -448,6 +449,12 @@ INSERT INTO step(step_name, type, action_id, data) VALUES ('map_question_respons
 INSERT INTO step(step_id, step_name, type, data) VALUES ('d1cbc4a8-ce4c-4734-8e71-a824d30c401a', 'edit_metadata_in_meditor_after_publication_form_review', 'action', '{"rollback":"send_to_meditor_after_publication_form_review","type": "action"}');
 INSERT INTO step(step_id, step_name, type, data) VALUES ('c628d63b-93b9-45ae-8e7b-a903554b6726', 'send_to_meditor_after_publication_form_review', 'action', '{"rollback":"map_question_response_to_ummc","type": "action"}');
 
+-- Unknown DAAC
+-- StepEdge(workflow_id, step_name, next_step_name)
+INSERT INTO step_edge VALUES ('3335970e-8a9b-481b-85b7-dfaaa3f5dbd9', 'init', 'data_accession_request_form');
+INSERT INTO step_edge VALUES ('3335970e-8a9b-481b-85b7-dfaaa3f5dbd9', 'data_accession_request_form', 'data_accession_request_form_review');
+INSERT INTO step_edge VALUES ('3335970e-8a9b-481b-85b7-dfaaa3f5dbd9', 'data_accession_request_form_review', 'assign_a_workflow');
```

Comment:
We should consider here how not ending on a close step might affect metrics which specifically look for reaching a close step to consider complete.

Reply:
If it gets reassigned to a daac its not complete  until it finishes there's and if it gets rejected it would move to withdrawn and never closes.  However we can add a close step to cover off potential edge casing.

Reply:
That's a good point. I think we'd want to capture the time it takes in the unknown workflow and being reassigned as well.
Though, we might need to consider a way of allowing rejected/withdrawn requests to `close` but add a way to differentiate between `close` meaning completed a workflow vs `close` meaning completed as much of the workflow as it ever will because it was rejected/withdrawn.

Reply:
We would probably need to change some of the time tracking logic to track time in unknown workflow as well as time in new workflow.

right now I think the time tracking would combine those to at completion of the second.

Reply:
I think that will still be valuable information i.e. maybe there's a need for additional support of reassigning unknown DAAC workflow which is causing a bottleneck.

Reply:
Can that be a follow on ticket to give time to think about the best way to handle that?

Reply:
Yes 👍